lib/obsolete.gd

```diff
@@ -76,9 +76,9 @@
 ##  <#/GAPDoc>
 ##
 
-BIND_GLOBAL( "DeclareObsoleteSynonym", function( name_obsolete, name_current, desc )
+BIND_GLOBAL( "DeclareObsoleteSynonym", function( name_obsolete, name_current )
```

Comment:
So, I understand this as that you're suggesting not to specify the version in which it may be deleted, just because it's not guaranteed to happen.